### DISCUSSION TITLE
[테스트 서버] root 비밀번호 설정

### DISCUSSION THREAD

**Discussion by Subak-Uncle** [cca-ffodregamdi/running-hi-back-v2#435] [Announcements]
현재 테스트 서버에 접속하면 명령어에 `sudo`를 붙여야 합니다.

이를 쓰지 않기 위해선 root 계정으로 로그인을 해야 하는데요!!

```shell
  # root 로그인 명령어
  su - 
```
패스워드를 입력하면 루트 계정으로 로그인이 되어 모든 명령어에 `sudo`를 붙이지 않아도 됩니다.
패스워드는 애플 디벨로퍼 팀 계정 비밀번호입니다^^

**Comment by numerical43** [comment]
정말 엄청나고 굉장하네요!! 굿입니다요👍

**Reply by Subak-Uncle** [reply]
꿍꿍꿍!
